src/sentry/tempest/tasks.py

```diff
@@ -227,6 +227,14 @@ def fetch_items_from_tempest(
         "attach_screenshot": attach_screenshot,
     }
 
+    logger.info(
+        "Tempest API request",
+        extra={
+            "payload": payload,
```

Comment:
How large is this payload? If it turns out truncated, you can add it as attachment instead: https://github.com/getsentry/sentry/blob/de4ab4e10d20dabb479ade61dadd03d49601228f/src/sentry/consumers/validate_schema.py#L48-L49

Reply:
Just 6 fixed size values so I hope it should be fine.